aptos-move/framework/aptos-token/sources/TokenCoinSwap.move

```diff
@@ -2,14 +2,14 @@
 /// 1. hold tokens escrow to prevent token been transferred
 /// 2. list token for swapping with a targeted CoinType.
 /// 3. execute the swapping
-module AptosFramework::TokenCoinSwap {
+module AptosToken::TokenCoinSwap {
```

Comment:
Minor changes:
/// 1. Hold tokens in escrow to prevent the token from being transferred.
/// 2. List token for swapping with a targeted CoinType.
/// 3. Execute the swapping.

Reply:
updated